test/OpenCensus.Tests/Impl/Stats/ViewManagerTest.cs

```diff
@@ -745,8 +723,9 @@ public void SettingStateToDisabledWillClearStats_Cumulative()
 
         private void SettingStateToDisabledWillClearStats(IView view)
         {
+            // TODO: deal with timestamp validation
```

Comment:
is this WIP?

Reply:
this comment is for the future. Test needs bigger refactoring which I'd suggest to do when we will enable code coverage tracking for the solution